src/hooks/a11y/useBodyScrollLock.js

```diff
@@ -0,0 +1,30 @@
+import { useLayoutEffect } from 'react'
+
+export const LOCK_SCROLL_STYLES = (scrollPosition) =>
+  `overflow: hidden; position: fixed; width: 100%; top: ${-scrollPosition}px;`
+
+/**
+ * Hook set's scroll lock on the body element
+ *
+ * @param {boolean} setBodyOverflow - flag set's if the body is currently scroll locked
+ *
+ * @return {void}
+ */
+function useBodyScrollLock(setBodyOverflow) {
```

Comment:
I'm realizing I've done a horrible job of exposing our hooks and validators!!

Would you mind trying to add this to the styleguide.config.js?

```js
    {
      name: 'Hooks',
      content: './src/hooks/hooks.md',
      components: './src/hooks/**/*.js',
      sectionDepth: 2,
    },
    {
      name: 'Validators',
      content: './src/validators/validators.md',
      components: './src/validators/**/*.js',
      sectionDepth: 2,
    },
```

and just put placeholders for the markdown files?